src/Monocle/Backend/Documents.hs

```diff
@@ -339,7 +339,7 @@ data EDocType
 
 allEventTypes :: [EDocType]
 allEventTypes =
-  filter (`notElem` [EChangeDoc, EOrphanTaskData, ECachedAuthor]) [minBound .. maxBound]
+  filter (`notElem` [EChangeDoc, EOrphanTaskData, ECachedAuthor, EErrorDoc]) [minBound .. maxBound]
```

Comment:
Should we do an inclustion test instead? Or maybe add a note in the EDocType definition?

Reply:
Sure I've added a note.